$OpenBSD: patch-wxPython_setup_py,v 1.5 2019/01/02 12:43:19 jca Exp $

Drop the building of the media extension to avoid the dependency
on wxWidgets-media

Index: wxPython/setup.py
--- wxPython/setup.py.orig
+++ wxPython/setup.py
@@ -424,21 +424,6 @@ ext = Extension('_html', swig_sources,
 wxpExtensions.append(ext)
 
 
-mediaLibs = libs[:]
-if not MONOLITHIC and findLib('media', libdirs):
-    mediaLibs += makeLibName('media')
-swig_sources = run_swig(['media.i'], 'src', GENDIR, PKGDIR,
-                        USE_SWIG, swig_force, swig_args, swig_deps)
-ext = Extension('_media', swig_sources,
-                include_dirs =  includes,
-                define_macros = defines,
-                library_dirs = libdirs,
-                libraries = mediaLibs,
-                extra_compile_args = cflags,
-                extra_link_args = lflags,
-                **depends
-                )
-wxpExtensions.append(ext)
 
 
 swig_sources = run_swig(['webkit.i'], 'src', GENDIR, PKGDIR,
@@ -1011,7 +996,7 @@ if __name__ == "__main__":
     if not PREP_ONLY:
 
         if not EGGing:
-            if INSTALL_MULTIVERSION:
+            if 1:
                 setup(name             = 'wxPython-common',
                       version          = VERSION,
                       description      = DESCRIPTION,
